sample_data_sf-0.01/supplier.tbl

```diff
@@ -0,0 +1,20 @@
+1|Supplier#000000001|sdrGnXCDRcfriBvY0KL,i|PERU     3|PERU|AMERICA|27-989-741-2988|
+2|Supplier#000000002|TRMhVHz3XiFu|ETHIOPIA 5|ETHIOPIA|AFRICA|15-768-687-3665|
+3|Supplier#000000003|BZ0kXcHUcHjx62L7CjZS|ARGENTINA3|ARGENTINA|AMERICA|11-719-748-3364|
+4|Supplier#000000004|qGTQJXogS83a7MB|MOROCCO  2|MOROCCO|AFRICA|25-128-190-5944|
+5|Supplier#000000005|lONEYAh9sF|IRAQ     0|IRAQ|MIDDLE EAST|21-750-942-6364|
+6|Supplier#000000006|zaux5FT|KENYA    1|KENYA|AFRICA|24-114-968-4951|
+7|Supplier#000000007| 0W7IPdkpWycU|UNITED KI0|UNITED KINGDOM|EUROPE|33-190-982-9759|
+8|Supplier#000000008|S8AWPqjYlanEQlcDO2|PERU     4|PERU|AMERICA|27-147-574-9335|
```

Comment:
`PERU 4`. The spaces and numbers after the country names look "strange". Is it generated on purpose like that?

Reply:
> The S_CITY column is created using the first 9 characters of the S_NATION (blank extended if there are fewer than 9) followed by a digit 0-9.

https://www.cs.umb.edu/~poneil/StarSchemaB.PDF p. 4